.circleci/config.yml

```diff
@@ -105,6 +105,25 @@ jobs:
           https://github.com/go-redis/redis
           contrib/go-redis/redis/vendor/github.com/go-redis/redis
 
+    - run:
+        name: Checkout k8s.io/client-go
+        # This step checks out k8s.io/client-go so we can
+        # vendor the dependencies.
+        command: >
+          git clone
+          https://github.com/kubernetes/client-go
+          $GOPATH/src/k8s.io/client-go
+
+    - run:
+        name: Vendor gnostic
+        # This step vendors github.com/googleapis/gnostic in the
+        # k8s.io/client-go package, because it won't build with the
+        # current master.
+        command: >
+          git clone --branch v0.4.0
```

Comment:
I chose the maximum, `0.4.0`, that is before the breaking change. Since go get by default checks out the most recent, this is the closest version to what was already being built.